apps/transport/lib/transport_web/controllers/resource_controller.ex

```diff
@@ -206,13 +195,20 @@ defmodule TransportWeb.ResourceController do
 
   @spec form(Plug.Conn.t(), map()) :: Plug.Conn.t()
   def form(conn, %{"dataset_id" => dataset_id} = params) do
+    conn =
+      conn
+      |> assign_or_flash(
+        fn -> Datasets.get(dataset_id) end,
```

Comment:
```suggestion
        fn -> Datagouvfr.Client.Datasets.get(dataset_id) end,
```

Pour ne pas confondre avec `DB.Dataset`.

Envisager de supprimer l'alias dans ce module